docs/topics/azure-stack.md

```diff
@@ -153,11 +158,23 @@ If you need to expose more than 5 services, then the recommendation is to route
 
 These are the Kubernetes versions that you can deploy to Azure Stack using AKS Engine:
 
+<!-- - 1.14.3 build and push to mcr -->
+<!-- - 1.13.7 build and push to mcr -->
+- 1.14.1
```

Comment:
I don't see `1.14.1-azs` in the VHD script. I can add it, assuming this is correct.

Reply:
you are the best, thx.